goentri.com.hotmart.json

```diff
@@ -0,0 +1,28 @@
+{
+  "providerId": "goentri.com",
+  "providerName": "Entri",
+  "serviceId": "hotmart",
+  "serviceName": "Hotmart",
+  "version": 1,
+  "logoUrl": "https://cdn.goentri.com/logo.svg",
+  "description": "Allows user to easily set up dkim record using Entri",
+  "syncPubKeyDomain": "goentri.com",
+  "syncRedirectDomain": "api.goentri.com, goentri.com, entri.com, app.entri.com,app.goentri.com",
+  "variableDescription": "primaryIp is the IP, cnamehost is the host for the CNAME and cnamePointsTo is the URL to where the subdomain is pointed ",
```

Comment:
and here subdomain is mentioned, the template looks like a root domain template with redirect.

Reply:
and here the subdomain means the subdomains the user is creating, www for example.